src/Console/Application.php

```diff
@@ -114,6 +119,12 @@ private function initializeApplication(InputInterface $input): array
                 $tasks[] = $function;
             } elseif ($function instanceof ContextDescriptor) {
                 $this->contextRegistry->add($function);
+            } elseif ($function instanceof ListenerDescriptor && null !== $function->reflectionFunction->getClosure()) {
```

Comment:
Original message from @lyrixx 
> Why do we need
> ```null !== $function->reflectionFunction->getClosure()```
> ?

That is required to comply with phpstan, because closure can be nullable